python/pyhail/dataset.py

```diff
@@ -266,6 +337,125 @@ def annotate_samples_list(self, input, root):
     def annotate_samples_table(self, input, sample_expr, root=None, code=None, config=None):
         """Annotate samples with delimited text file (text table).
 
+        **Examples**
+
+        >>> conf = pyhail.TextTableConfig(impute=True)
+        >>> vds1 = (hc.read('data/example1.vds')
+        >>>  .annotate_samples_table('data/samples1.tsv', 'Sample', root='sa.phenotypes', config=conf)
+        >>>  .print_schema())
```

Comment:
print_schema doesn't (shouldn't) return a vds.  In general, I wouldn't include print_schema in your examples (unless you're documenting print_schema).  Also, why example1?  We should probably have a single, simple example.vds that can be used for all examples.